common/app/views/fragments/header.scala.html

```diff
@@ -5,132 +5,176 @@
-        role="banner"
-        data-link-name="global navigation: header">
-    <div class="js-navigation-header navigation-container navigation-container--collapsed">
-        <div class="gs-container l-header__inner">
-            <div class="l-header-pre u-cf">
-                <div class="brand-bar">
-                    @if(Edition(request) == Uk) {
-                        <div class="brand-bar__item brand-bar__item--profile
-                                    brand-bar__item--has-control brand-bar__item--register
-                                    popup-container has-popup js-profile-register
-                                    hide-until-mobile-landscape">
-                            <a href="@Configuration.id.membershipUrl?INTCMP=DOTCOM_HEADER_BECOMEMEMBER"
-                                data-link-name="Register link"
-                                class="brand-bar__item--action">
-                                    @fragments.inlineSvg("profile-add-36", "icon", List("rounded-icon", "control__icon-wrapper"))
-                                <span class="brand-bar__item__badge control__icon-wrapper control__icon-wrapper--text">free</span>
-                                <span class="control__info js-control-info control__info--supporting">become a member</span>
-                            </a>
-                        </div>
-                    }
-                    @if(IdentityProfileNavigationSwitch.isSwitchedOn) {
-                        <div class="brand-bar__item brand-bar__item--profile popup-container
-                                    has-popup js-profile-nav"
-                             data-component="identity-profile">
-                             <a href="@Configuration.id.url/signin?INTCMP=DOTCOM_HEADER_SIGNIN"
-                               data-link-name="User profile"
-                               data-toggle="popup--profile"
-                               data-toggle-signed-in="true"
-                               class="brand-bar__item--action popup__toggle"
-                               data-test-id="sign-in-link"
-                               aria-haspopup="true">
-                                @fragments.inlineSvg("profile-36", "icon", List("rounded-icon", "control__icon-wrapper"))
-                                @if(conf.switches.Switches.SaveForLaterSwitch.isSwitchedOn) {
-                                    @fragments.inlineSvg("bookmark", "icon", List("save-for-later__icon"))
-                                }
-                                <span class="js-profile-info control__info"
-                                      data-test-id="sign-in-name">sign in</span>
-                            </a>
-                            <div class="js-profile-popup">
-                                <ul class="popup popup--default popup__group popup--profile is-off" data-link-name="Sub Sections" data-test-id="popup-profile">
+@if(mvt.ABNewHeaderVariant.isParticipating) {
```

Comment:
consider separating into two files rather than a big if statement, makes it easier to see which is new